go/ct/st/accounts.go

```diff
@@ -32,7 +32,7 @@ func NewAccounts() *Accounts {
 
 func (a *Accounts) GetCodeHash(address tosca.Address) (hash [32]byte) {
 	hasher := sha3.NewLegacyKeccak256()
-	hasher.Write(a.code[address].ToBytes())
+	_, _ = hasher.Write(a.code[address].ToBytes())
```

Comment:
Like for the random number generator, we should comment that the hasher is never producing an error (assuming this is true):
```suggestion
	_, _ = hasher.Write(a.code[address].ToBytes()) // hasher never produces an error
```

Reply:
yes, 
from the Hash interface
```go
type Hash interface {
	// Write (via the embedded io.Writer interface) adds more data to the running hash.
	// It never returns an error.
	io.Writer
```
the Read method is a little more complicated, we declare an interface in our own namespace 
```go
// keccakState wraps sha3.state. In addition to the usual hash methods, it also supports
// Read to get a variable amount of data from the hash state. Read is faster than Sum
// because it doesn't copy the internal state, but also modifies the internal state.
type keccakState interface {
	hash.Hash
	Read([]byte) (int, error)
}
```
This interface "discovers" a method Read implemented in crypto@xxx/sha3, the body never returns an error.